examples/asrc_demo/src/usb/dbcalc.c

```diff
@@ -0,0 +1,121 @@
+// Copyright 2023 XMOS LIMITED.
+// This Software is subject to the terms of the XMOS Public Licence: Version 1.
+
+#include <xs1.h>
+#include <stdint.h>
+/* The coefficients of the chebychev polynomial to approximate 10^x in the interval [-1,1].
+   This polynomial was calculated using the mpmath library in Python:
+
+   from mpmath import *
+   mp.dps = 15
+   mp.pretty = True
+   poly, err = chebyfit(lambda x: pow(10,x), [-1,1], 14, error=True)
+   nprint([int(x * pow(2,28)) for x in poly])
+ */
+#define COEF_PREC 28
+static uint32_t coef[14] = {2407, 13778, 64588, 308051, 1346110, 5261991, 18277531, 55564576, 144789513, 314406484, 546179875, 711608713, 618095479, 268435456};
+
+#define DB_CALC_PREC 28
+
+/* Function: db_to_mult
+
+     This function converts decibels into a volume multiplier. It uses a fixed-point polynomial approximation
+     to 10^(db/10).
+
+   Parameters:
+       db               - The db value to convert.
+       db_frac_bits     - The number of binary fractional bits in the supplied decibel value
+       result_frac_bits - The number of required fractional bits in the result.
+
+   Returns:
+       The multiplier value as a fixed point value with the number of fractional bits as specified by
```

Comment:
The documentation of this function doesn't seem to use the same style as the other functions.

Reply:
I know why this is - it's borrowed wholesale from lib_xua. It is documented at least..